src/django/contricleaner/lib/helpers/is_valid_type.py

```diff
@@ -0,0 +1,8 @@
+def is_valid_type(value):
```

Comment:
Could you please add type hints?

Reply:
Done

Reply:
Do we have type `any` in Python?
I think you should use https://docs.python.org/3/library/typing.html#the-any-type